handlers/main.yml

```diff
@@ -13,3 +7,11 @@
 - name: load-templates
   include: ./handlers/elasticsearch-templates.yml
   when: es_templates
+
```

Comment:
This needs to occur before the load templates i think - for the reasons state in the above comment.